### DISCUSSION TITLE
The extension couldn't load on this page. (ERROR MESSAGE)

### DISCUSSION THREAD

**Discussion by AvignonA23** [GorvGoyl/ChatGPT-Writer#21] [Report Bugs and Issues]
The extension couldn't load on this page.

If the tab was opened before the extension was installed, then please reload the tab once to use the extension.

Please also note that extension can not run on special pages like browser UI pages, extension pages, and Chrome web store (chrome.google.com).
If issue persists, please [report it here](https://github.com/GorvGoyl/ChatGPT-Writer/discussions/categories/report-bugs-and-issues).
![Screenshot 2023-01-16 at 2 51 42 PM](https://user-images.githubusercontent.com/122800861/212693879-4d9f37e4-5230-4352-8d43-c387189d0cb7.png)

**Comment by GorvGoyl** [comment]
which website are you referring to?

**Reply by AvignonA23** [reply]
Hi Gourav: This is the chrome extension : ChatGPT Writer - Write mail, messages with AI
See screenshots below 
![Screenshot 2023-01-17 at 9 17 21 AM](https://user-images.githubusercontent.com/122800861/212844926-37a45fc2-83a0-4006-a7b3-0b4f2c2953ae.png)
![Screenshot 2023-01-17 at 9 17 59 AM](https://user-images.githubusercontent.com/122800861/212844930-e7850789-4cb5-4b0e-be0e-f0a06e09d593.png)

**Comment by GorvGoyl** [comment]
I meant on which website do you see that error?

**Reply by AvignonA23** [reply]
It is from the Google chrome store, I installed the chrome extension, and then when I want to use it I get that error. Can you try to install it and try from your side? 
![Screenshot 2023-01-17 at 9 17 59 AM](https://user-images.githubusercontent.com/122800861/212845711-0a6c6cbf-1797-4667-9eed-f7ab339ffb46.png)

**Comment by GorvGoyl** [comment]
can you share the full screenshot of the error along with the website url

**Comment by GorvGoyl** [comment]
it should work on other sites. Try visiting some site and launch the
extension :)

**Comment by anewbies** [comment]
Se estiver usando o Google Chrome, entre na configuração chrome://settings/syncSetup e marque a caixa on "verificação de ortografia aprimorada" e o problema estará resolvido.
![image](https://user-images.githubusercontent.com/14927641/213434388-b31201f0-4ac8-4fcc-a0eb-626e83ff2ef3.png)
